lib/utils.pm

```diff
@@ -1959,4 +1960,8 @@ sub assert_secureboot_status {
     die "Error: SecureBoot is $statestr" if $state xor $expected;
 }
 
+sub is_opensuse_performance {
+    return (is_opensuse && get_var('SOFTFAIL_BSC1063638') && get_var('ROLLBACK_AFTER_MIGRATION'));
+}
+
```

Comment:
A file with nearly 2k lines is realistically unmaintainable. I suggest to not extend the file

Reply:
OK, removed.

Reply:
I don't think it is better to remove it directly, you'd better to move it to another file. @okurz How do you think move it to lib/main_common.pm instead?